root/entity/Subscribers.js

```diff
@@ -76,32 +101,68 @@ const Subscribers = ({
           </ul>
         </>
       ) : (
-        <p>
-          {texp.l('There are currently no users subscribed to {entity}.',
-            {entity: entity.name})}
-        </p>
+        entityType === 'editor' ? (
+          viewingOwnProfile ? (
+            <p>
```

Comment:
Nitpicking: `p` element could actually precede the those two above conditions, thus needing less lines of code. The same change could be made with most other `p` elements of this PR, in a separate commit if needed for clarity.

Reply:
Done

Reply:
And I forgot to push the commit, lol. Well, I'll push it after looking at @mwiencek's comments.

Reply:
Thank you for https://github.com/metabrainz/musicbrainz-server/pull/962/commits/dba82eab57d6ce1afbdb96c26f3abda10a4fc4db